src/server/services/generate-arpa-report.js

```diff
@@ -2,11 +2,11 @@
 const path = require('path')
 const { mkdir, rmdir, writeFile, readdir, readFile } = require('fs/promises')
 const moment = require('moment')
-const { stringify } = require('csv-stringify/sync')
 const zipper = require('zip-local')
 const XLSX = require('xlsx')
 
 const { applicationSettings } = require('../db/settings')
+const { documentsWithProjectCode } = require('../db')
```

Comment:
hmmm is there a more specific place to require this from? i'm not sure how i feel about re-exporting so much stuff in the db index